.github/workflows/remove-experimental.yml

```diff
@@ -52,6 +52,11 @@ jobs:
           popd
       # remove the regulations site
       - name: remove experimental regulations site server
+      # If previous step fails the database will not be removed and you cannot rerun it
+      # In this instance we want to just remove the database so that we dont have 
+      # to go into the database and do it manually.
+      # If process is manually stopped it will not run this step.
+        if: success() || failure()
```

Comment:
Doesn't this negate each-other? I mean the process is success or a failure. What is the purpose of the if condition?

Reply:
Manual cancellation.  So if for whatever reason I close the pr but I'm like "dont do it no", and cancel it, it wont run the step.